test/test_parser.py

```diff
@@ -73,13 +76,20 @@ def testMtlTextureName(self):
         self.assertEqual(self.material1.texture.image_name,
                          prepend_dir('4x4.png'))
 
+
 class TestParserFailure(unittest.TestCase):
 
     def testMissingParseFunction(self):
         "Attempting to parse with a missing parse function should raise an exception."
-        # since no parse functions have been defined, this will always fail
-        self.assertRaises(Exception, pywavefront.parser.Parser, prepend_dir('uv_sphere.obj'))
+        # since no parse functions have been defined, this will always fail in strict mode
+        file_name = 'simple.obj'
+        parser = pywavefront.parser.Parser(prepend_dir(file_name))
+        pywavefront.parser.Parser.strict = True
+        self.assertRaises(AttributeError, parser.read_file, prepend_dir(file_name))
+        pywavefront.parser.Parser.strict = False
 
     def testMissingParsedFile(self):
         "Referencing a missing parsed file should raise an exception."
-        self.assertRaises(Exception, pywavefront.parser.Parser, 'missing.file.do.not.create')
+        file_name = 'donotexsit.obj'
```

Comment:
irrelevant typo in the file name.

Reply:
I typo'ed filename that doesn't exist will definitely not exist!